src/collections/_documentation/apm/distributed-tracing.md

```diff
@@ -0,0 +1,340 @@
+---
+title: Distributed Tracing
+sidebar_order: 1
+---
+
+[Alert Box: What plan APM is available on? ]
+
+Enabling tracing data collection can augment your existing errors to answer any question or unknowns about your systems. Tracing also allows you to track critical metrics and service-level objectives (SLO) like throughput, latency, or apdex to build a complete monitoring solution to diagnose and measure your application health.
+
+## Tracing & Distributed Tracing
+
+Applications (for example, web applications) typically consist of interconnected applications/components (also called services). For instance, let us assume a modern web application could be composed of the following components separated by the network boundaries:
+
+- Frontend (Single-Page Application)
+- Backend
+- Search Service
+- Cache Service
+
+Each of these components may be written in different languages on different platforms. Today, all of these components can be instrumented with the Sentry SDKs to capture error data or crash reports whenever an event occurs in any one of these components.
+
+With tracing, we can follow the journey of the API endpoint requests from their source (for example, from the frontend), and instrument code paths as these requests traverse each component of the application. This journey is called a **trace**. Traces that cross between components, as in our web application example, are typically called **distributed traces**.
+
+![DON%20T%20Add%20content%20to%20this%20FINAL%20Draft%20Tracing%20for%20/Tracing-diagram.png](DON%20T%20Add%20content%20to%20this%20FINAL%20Draft%20Tracing%20for%20/Tracing-diagram.png)
+
+Traces that start with the same set of API endpoint calls can potentially differ in terms of what code paths were executed. For example, an initial trace may result in a cache miss at the Cache Service, and a subsequent trace may include more information instrumented at the Cache Service. Although these two traces are similar, they are not identical.
+
+Each trace has a marker called a `trace_id` . Trace IDs are pseudorandom fixed length of alphanumeric character sequences.
+
+By collecting traces of your users as they use your applications, you can begin to reveal some insights such as:
+
+- Investigate and debug what occurred for a specific error event, or issue
+- What conditions are causing bottlenecks or latency issues in the application
+- Which endpoints or operations consume the most time.
+
+### Trace propagation model
+
+Referring back to our earlier example of the web application consisting of these high-level components:
+
+- Frontend (Single-Page Application)
+- Backend
+- Search Service
+- Cache Service
+
+A trace propagates first from the frontend, then the backend, and later to the search or caching service. Collected spans from each component are sent back to Sentry asynchronously and independently. Instrumented spans received from one component are not forwarded to the next component.
+
+## Span
```

Comment:
Why don't we just link to the glossary here?

Reply:
Yep! That's on my To Do. ;)